packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -1740,4 +1663,226 @@ describe("Garbage Collection Tests", () => {
 		assert.strictEqual(garbageCollector.tombstones.length, 0, "Expecting 0 tombstone nodes");
 		assert.strictEqual(garbageCollector.deletedNodes.size, 0, "Expecting 0 deleted nodes");
 	});
+
+	describe("GC stats", () => {
+		let garbageCollector: IGarbageCollector;
+		let initialStats: IGCStats;
+
+		const tests = (sweepEnabled: boolean) => {
+			beforeEach(() => {
+				// Set up initial GC graph with 5 nodes and 2 are unreferenced.
```

Comment:
```suggestion
				// Set up initial GC graph with 4 nodes and 2 are unreferenced.
```

Reply:
Or are you considering the "/" a node too? I guess it is

Reply:
There are 5 nodes including "/"